src/types.ts

```diff
@@ -279,12 +279,12 @@ export type SuspendedNonDet = Omit<Suspended, 'status'> & { status: 'suspended-n
   value: Value
 }
 
-export interface SuspendedEcEval {
-  status: 'suspended-ec-eval'
+export interface SuspendedCSEval {
```

Comment:
A very minor nitpick, but is this supposed to be `SuspendedCseEval`, not `SuspendedCSEval`?

Reply:
Yes, I agree. It should be SuspendedCseEval.

The status string should be 'suspended-cse-eval'.